scripts/build.sh

```diff
@@ -41,11 +41,21 @@ export FM_ELECTRS_DIR="$FM_TEST_DIR/electrs"
 mkdir -p $FM_CLN_DIR
 mkdir -p $FM_LND_DIR
 mkdir -p $FM_BTC_DIR
+cat <<EOF > $FM_BTC_DIR/bitcoin.conf
```

Comment:
I don't get it. I thought the goal is to point with `-c` at the existing file, and here are we create a new one, which makes it the same as `cp`, just from an inline source?

Reply:
The directory of Existing bitcoin.conf file is not same as that of the fixtures.rs hence we can't directly add "-conf=" at the place where we are spawning the bitcoind. Also in the issue it was asked to get rid of the cp command and instead pass the full path . This the way I could think of to fix it.